docs/src/main/asciidoc/infinispan-client-guide.adoc

```diff
@@ -99,15 +103,93 @@ public class Book {
 }
 ----
 
-The default serialization is done using a library based on protobuf. We need to define the proto
-buf schema and a marshaller for each user type(s).
+Serialization of user types uses a library based on protobuf, called Protostream.
 
-NOTE: Annotation based proto stream marshalling is not yet supported in
-the Quarkus Infinispan client. This will be added soon, allowing you to only annotate your classes,
-skipping the following steps.
+=== Annotation based Serialization
 
-NOTE: This version has an issue currently requiring the Query DSL module as a dependency when running in
-native mode with proto buf based marshalling. This should be remedied in a future version.
+This can be done automatically by adding annotations to your user classes from the protostream library
```

Comment:
This sentence is unclear: `annotations to your user classes from the protostream library` .

Looks like the user classes are from the Protostream library, whereas I think you meant the annotations?

Reply:
Sure, I can cleanup.